packages/itinerary-body/src/styled.tsx

```diff
@@ -430,20 +440,21 @@ export const MapIcon = styled(Map).attrs(props => ({
 }))``;
 
 export const PlaceDetails = styled.div`
-  /* container for Leg details */
-  /* padding: 15px 0 15px 15px; */
-  /* padding-bottom: 15px; */
+  grid-row-start: 2;
+  grid-column-start: 3;
 `;
 
 export const PlaceHeader = styled.div`
   display: flex;
   font-size: 1.2em;
 `;
 
-export const PlaceName = styled.div`
+export const PlaceName = styled.h2`
```

Comment:
If each place is an `<li>`, this can be just `<span>`.

Reply:
A place would not really be part of the "Table of contents" of the page, right?

Reply:
You're right!